### ISSUE TITLE
Remove stage 0 Babel features

### ISSUE THREAD

**Issue by herrstucki** [interactivethings/catalog#48]
Set .babelrc to `"stage": 2` and fix everything until it works again.

**Comment by herrstucki** [comment]
For example, something like

```js
class Foo extends React.Component {
  static propTypes = {}
  render() {}
}
```

should be rewritten to

```js
class Foo extends React.Component {
  render() {}
}

Foo.propTypes = {}
```

**Comment by tpreusse** [comment]
Why? `Foo.propTypes = {};` is so ugly!

**Comment by herrstucki** [comment]
It's not about ugliness. It's about writing code that will work in the future.

I read (unfortunately I don't remember where anymore) that class properties are still disputed and it's uncertain if they will make it into the spec. Same goes for decorators and function binding with `::`.

In any case, I think it's good practice to use as few experimental language features as possible. Stage 2 is a good compromise.

**Comment by bldng** [comment]
Resolved with 82cdd6a
